crates/virtio-queue/LICENSE-APACHE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think we decided that we don't necessarily need to have these files in the repositories as long as the licenses are mentioned where they have to. Maybe we should update the docs if that's the case.

Reply:
Cool, I've removed them from the PR and will update the docs separately.